configurator/src/main.rs

```diff
@@ -163,6 +177,38 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let cli = Cli::parse();
 
     match cli.command {
+        Commands::Difference {
+            pictures,
+            output_dir,
+        } => {
+            let before = opencv::imgcodecs::imread(
+                pictures[0].to_str().expect("Path wasn't valid unicode"),
+                1,
+            )?;
+            let before = cv::Picture::from(before);
```

Comment:
How about creating a `cv::Picture::load` constructor to about duplication + not rely on OpenCV directly outside of cv.rs, so that switching away from OpenCV later will be easier?

Reply:
Good point. Done.